lib/matplotlib/tests/test_ticker.py

```diff
@@ -169,7 +169,7 @@ def test_SymmetricalLogLocator_set_params():
     nose.tools.assert_equal(sym.numticks, 8)
 
 
-@cleanup
+@cleanup(style='classic')
```

Comment:
Yes for image_comparison, but no for cleanup.  It looks to me like that is a bug in cleanup.